module/zfs/spa.c

```diff
@@ -9309,9 +9313,11 @@ EXPORT_SYMBOL(spa_event_notify);
 #endif
 
 #if defined(_KERNEL)
-module_param(spa_load_verify_maxinflight, int, 0644);
-MODULE_PARM_DESC(spa_load_verify_maxinflight,
-	"Max concurrent traversal I/Os while verifying pool during import -X");
+/* BEGIN CSTYLED */
+module_param(spa_load_verify_shift, int, 0644);
+MODULE_PARM_DESC(spa_load_verify_shift, "log2(fraction of arc that can "
```

Comment:
I think in ZoL all the tunables are listed in the zfs-params man page, so I think you may have to change the name of the tunable on that file too (or introduce it if it doesn't exist).

Reply:
Fixed